hudi-flink/src/main/java/org/apache/hudi/sink/partitioner/BucketAssignFunction.java

```diff
@@ -125,12 +126,18 @@
    */
   private final boolean globalIndex;
 
+  private final double ttlTime;
+
+  private final SimpleDateFormat instantDateFormat;
```

Comment:
how about use `org.apache.hudi.common.table.timeline.HoodieActiveTimeline#COMMIT_FORMATTER` to replace `instantDateFormat ` to avoid import `SimpleDateFormat ` again ?

Reply:
SimpleDateFormat not thread safe